try:
    a=2
    b=4
    print(a+b)

except exception as e:
    print(e)
